CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 ## CHANGE LOG
 
+### 0.7.0 - QA v1
+- This was the first of two sprints before we go from Internal Beta to Public Beta. YAY!  :rocket: :books:
+- A lot of accessibility, QA, and feedback fixes were completed this sprint.
+  - Each page was checked through a11y tools.
+  - Page titles, duplicate IDs, color contrast, valid HTML, and other fixes were completed.
+- The EDD form went through another iteration with updated content, input validation, and error checking.
+  - This includes listing the errors and making them links to the input fields.
+- Also not allowing to place a request if EDD is not available or if there are no delivery locations.
```

Comment:
Thanks! Some changes we can add in addition,

1. We have default location as EDD or the first delivery location now
2. The links back to start over and back to search results.
3. Complete a whole no js search to place a hold process.
4. Suggestion texts of the search results in different fields.
5. Error handling for hold request page.